### DISCUSSION TITLE
Contributions to question with closed issues

### DISCUSSION THREAD

**Discussion by sotoblanco** [freeCodeCamp/Developer_Quiz_Site#654] [Q&A] [answered]
I saw questions with closed issues, such as Git or Python. Does this mean that contributions to files with open issues are preferential to questions with closed issues?

**Comment by jdwilkin4** [comment, marked as answer]
> Does this mean that contributions to files with open issues are preferential to questions with closed issues?

Yes that is correct.

**Reply by sotoblanco** [reply]
How about quiz recently added like DevOps and cloud computing? Are we looking preferential contribution on those as well or just the ones with open issues?

**Reply by jdwilkin4** [reply]
There is an existing list of cloud computing questions that we are reviewing and pulling the questions from. 
As for DevOps, we are waiting to hear back from the Learn to Code RPG team on if they are looking to add more questions for version 2 of the game. 

In general, the open issues are the ones were are still looking questions for.
Contributors are still free to add questions for other categories like Python or Git but we are not really pushing for those at this moment.

**Reply by sotoblanco** [reply]
Yes! Makes sense. Thanks a lot for your support!
